### PR TITLE
#163221709 Setup directory structure

### PR DESCRIPTION
**Pull request by tirgei**
#### What does this PR do?
This Pull Request creates the directory structure for version 2 of the API

#### Description of Task to be completed?
The tasks for this PR is:
- Create the modules
- Add a ```__init__.py``` file to each module
- Add the main run file
- Add the requirements file

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/Questioner-API.git ```

CD into the project folder on your machine
```$ cd Questioner-API ```

Check out the branch for this PR
``` $ git checkout  ch-setup-directory-structure-163221709``

#### Pivotal Tracker story associated with this PR
[#163221709](https://www.pivotaltracker.com/story/show/163221709)

#### Screenshots
directory2

directory2-tree

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,73 @@
+aniso8601==4.0.1
```
Comment (Duncanian) [inline requirements.txt]: The `requirements.txt` should be empty as per now as you should avoid having dependencies that are not being used.

**Review by Duncanian — CHANGES_REQUESTED** [review]
Kindly refactor

**Comment by Duncanian** [comment]
Instead of having the folder `v2`, move everything from the `v2` folder and put them in the `api` folder and delete the `v2` folder.

File: requirements.txt (hunk 1 above)
Comment (tirgei) [reply requirements.txt]: Noted

**Comment by tirgei** [comment]
> Instead of having the folder `v2`, move everything from the `v2` folder and put them in the `api` folder and delete the `v2` folder.

Okay.. let me refactor that

**Pull request merged**